app/controllers/api/spots_controller.rb

```diff
@@ -0,0 +1,13 @@
+class Api::SpotsController < ApplicationController
+  def create
+    spot = Spot.create!(create_params)
+
+    render json: spot
+  end
+
+  private
+
+  def create_params
+    params.require(:spot).permit(:skater_id, :lat, :long, :name)
```

Comment:
Pull skater_id out of this params.permit and put it in a tap like you did in the other controller